### ISSUE TITLE
Amadeus - Google app not working

### ISSUE THREAD

**Issue by Azuhiika** [Yink/Amadeus#90]
Hello! I recently installed the latest amadeus app. I saw some video where you tap kurisu to talk. Apparently when i tap, then tell her something, nothing happens. Is there a fix to this?

**Comment by tuodekun** [comment]
try setting Google as the default app of voice recognition and see if it works|•ω•`)

**Comment by Azuhiika** [comment]
Hi @tuodekun! I tried doing that, but i cannot see any option of doing so T_T My phone is running in Android Lollipop.

**Comment by tuodekun** [comment]
hmm......please forgive my poor English, but i dont get it by 'cannot see any option of doing so' . Does that mean that you cant find the place to change this default setting?(つд⊂)

**Comment by Azuhiika** [comment]
Yes I can't. Can you show me how? I have seen tips on how to do it but I don't think it is for Android Lollipop

**Comment by tuodekun** [comment]
no idea of lollipop since im using android 8.0😂try finding under 'applications' may do some good
ps:dont you think getting a more recent version of android is better?|•ω•`)
